config/examples/Creality/Ender-5 Pro/CrealityV422 BLtouch Micro-Swiss Direct Drive/README.md

```diff
@@ -0,0 +1,51 @@
+# Configuration files for Stock Ender 5 Pro 4.2.2 Board
+
+This configuration has some tweaks to jerk and acceleration. If you don't want those changes then set these values in the `Configuration.h` file for the following commands.
+
+- Line 1062: `#define DEFAULT_XJERK 10.0`
```

Comment:
Line numbers will change, so it’s best to not reference them directly.

Reply:
Good point. I removed those references, updated some instructions for clarity, and added the boot screen and status screen files.